src/System.Private.Uri/tests/FunctionalTests/UriRelativeResolutionTest.cs

```diff
@@ -136,6 +136,16 @@ public void Uri_Relative_BaseVsSimplePartialPathWithScheme_ReturnsPartialPathWit
         }
 
         [Fact]
+        public void Uri_Relative_SimplePartialPathWithUnknownScheme_Unicode_ReturnsPartialPathWithScheme()
+        {
+            string schemeAndRelative = "scheme:\u011E";
+            Uri resolved = new Uri(schemeAndRelative, UriKind.RelativeOrAbsolute);
+
+            String expectedResult = schemeAndRelative;
+            Assert.Equal(expectedResult, resolved.ToString());
```

Comment:
I agree with your suggestion, although I followed the convention in the file to have a `String expectedResult` variable and non-const const strings.